core/src/com/po/fuck/view/classdrawers/factories/RoomDrawerFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.po.fuck.view.classdrawers.factories;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.po.fuck.view.CenterDrawer;
+import com.po.fuck.view.Renderer;
+import com.po.fuck.view.classdrawers.ClassDrawer;
+import com.po.fuck.model.Room;
+
+public class RoomDrawerFactory {
+    
+    static {
+        Renderer.addDrawer(Room.class, RoomDrawerFactory.get(Room.class, new Sprite(new Texture("island2.png"))));
+    }
+
+    @SuppressWarnings("all")
```

Comment:
Why is it here ?

Reply:
Because for some reason Room is a final class, and it gives a warning for extending the bounded class. I think this is thinking in advance for future Room system development.